cpp/log/log_verifier.h

```diff
@@ -44,6 +44,8 @@ class LogVerifier {
         return "Inconsistent timestamps.";
       case INVALID_SIGNATURE:
         return "Invalid signature.";
+      case INVALID_MERKLE_PATH:
+        return "Invalid Merkle path.";
```

Comment:
Yeah, in this particular case a harmless default would have been nice.  But still at least the assertion failure alerted me to this missing case.